docs/source/kedro_project_setup/settings.md

```diff
@@ -0,0 +1,31 @@
+# Project settings
+
+A Kedro project's `settings.py` file contains the runtime configuration for the project, including registration of hooks and library components. This page explains how settings work and which settings are available.
+
+A project does not _need_ to have a settings file – in fact, by default, all code in `settings.py` is commented out. In the case that settings are not supplied, Kedro chooses sensible default values. You only need to edit `settings.py` if you wish to change to values other than the defaults.
+
+## Available settings
+
+```eval_rst
++-----------------------------------+---------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------+
+| Setting                           | Default value                                           | Use                                                                                                                                         |
++===================================+=========================================================+=============================================================================================================================================+
+| :code:`HOOKS`                     | :code:`tuple()`                                         | Instantiate and list your `project hooks <../extend_kedro/hooks.md>`_.                                                                      |
++-----------------------------------+---------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------+
+| :code:`DISABLE_HOOKS_FOR_PLUGINS` | :code:`tuple()`                                         | List the installed plugins for which to disable auto-registry.                                                                              |
++-----------------------------------+---------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------+
+| :code:`SESSION_STORE_CLASS`       | :code:`kedro.framework.session.session.BaseSessionStore`| Define where to store data from a KedroSession.                                                                                             |
++-----------------------------------+---------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------+
+| :code:`SESSION_STORE_ARGS`        | :code:`{ }`                                             | Define keyword arguments to be passed to `SESSION_STORE_CLASS` constructor.                                                                 |
```

Comment:
I think you did this because I suggested it but actually I don't like it 😅  Sorry!!
```suggestion
| :code:`SESSION_STORE_ARGS`        | :code:`dict()`                                             | Define keyword arguments to be passed to `SESSION_STORE_CLASS` constructor.                                                                 |
```

Reply:
Also does `SESSION_STORE_CLASS` render ok here or do you need to do `:code:`SESSION_STORE_CLASS`? Similarly for `CONFIG_LOADER_CLASS`.